ports/daxa/vcpkg.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "daxa",
+  "version-semver": "1.0.0",
+  "description": "Daxa C++ Vulkan Abstraction",
+  "homepage": "https://github.com/Ipotrick/Daxa",
```

```suggestion
  "homepage": "https://github.com/Ipotrick/Daxa",
  "license": "MIT",
```
A license is required.